library/report_database.inc

```diff
@@ -292,11 +292,35 @@ function collectItemizedRuleDisplayTitle($report_id, $itemized_test_id, $numerat
     $report_view = collectReportDatabase($report_id);
     $type_report = $report_view['type'];
     $dataSheet = json_decode($report_view['data'], true);
+    $display_group_provider_info = false;
+    $group_provider_label = '';
+
+    // for our group calculation reports we want to display the group and the provider label information
+    if ($report_view['provider'] == 'group_calculation') {
+        $display_group_provider_info = true;
+    }
     foreach ($dataSheet as $row) {
+        if ($display_group_provider_info) {
+            if (isset($row['is_provider_group'])) {
+                $group_provider_label = xl('Group') . ': ' . $row['name'] . ' ( ' . xl('TIN') . ' '
+                    . $row['federaltaxid'] . ' )';
+            } else if (isset($row['is_provider'])) {
+                $provider_label = ' ' . xl('Provider') . ': ' . $row['prov_fname'] . ',' . $row['prov_lname']
```

Comment:
need to html escape this,
xlt text text xlt text

Reply:
Good point and not sure why i did escaping in this function rather than on the echo of it (maybe i was using html in the past which have since removed). In that case would be best to strip escaping in the other items in this function.

Reply:
I was wrong here, this is escaping into the patient_finder and not into the cqm output so I do need to escape these values.